docs/content/en/docs/reference/baremetal/bare-prereq.md

```diff
@@ -3,10 +3,78 @@ title: "Requirements for EKS Anywhere on Bare Metal"
 linkTitle: "Requirements"
 weight: 15
 description: >
-  Preparing a VMware vSphere provider for EKS Anywhere
+  Bare Metal provider requirements for EKS Anywhere
 ---
 
-ADD CONTENT TO DESCRIBE:
+To run EKS Anywhere on Bare Metal, you need to meet the hardware and networking requirements described below.
 
-* Hardware requirements (IPMI, extra upgrade machines, etc.), including list of validated hardware models/families, etc.
-* Network requirements
+
+## Administrative machine
+
+Set up an Administrative machine as described in [Install EKS Anywhere ]({{< relref "../../getting-started/install/" >}}).
+
+## Computer requirements 
+
+The minimum number of physical computers needed to run EKS Anywhere in a non-production mode is:
+
+* Control plane physical machines: 1
+* Worker physical machines: 1
+
+The recommended number of physical machines for production is at least:
+
+* Control plane physical machines: 3
+* Worker physical machines: 2
+
+You will need an additional, temporary machine for each control plane node grouping and worker node grouping later when you go to upgrade a node.
+That machine must have the same specs as all the machines in that group.
+This comes from the need to use the same template to populate data on the disks for all nodes in a group.
+
+The computer hardware you need for your Bare Metal cluster must meet the following capacity requirements:
+
+* CPU: 2
+* Memory: 8GB RAM
+* Storage: 25GB
+
+## Network requirements
+
+Each machine should include the following features:
+
+* Network Interface Cards: At least one NIC is required. It must be capable of netbooting from PXE. 
+* IPMI integration (recommended): An IPMI implementation (such a Dell iDRAC or HP iLO) on the computer's motherboard or on a separate expansion card. This feature is used to allow remote management of the machine, such as turning the machine on and off.
+
+>**_NOTE:_** IPMI is not required for an EKS Anywhere cluster. However, without IPMI, upgrades are not supported and you will have to physically turn machines off and on when appropriate.
+
+Here are other network requirements:
+
+* All EKS Anywhere machine, including the Admin, control plane and worker machines, must be on the same layer 2 connection to the other machines in the cluster and have network connectivity to the BMC (IPMI, Redfish, and so on). The hardware does not need to be on the same layer 2 as the BMC, but the Admin machine and management cluster does need routes configured so it can communicate with the BMC API.
```

Comment:
```suggestion
* All EKS Anywhere machines, including the Admin, control plane and worker machines, must be on the same layer 2 network and have network connectivity to the BMC (IPMI, Redfish, and so on). The hardware does not need to be on the same layer 2 as the BMC, but the Admin machine and management cluster does need routes configured so it can communicate with the BMC API.
```

Reply:
Done.